### ISSUE TITLE
I wonder if you can add pictures of the characters 

### ISSUE THREAD

**Issue by ThanksILoveYouWithAllTheRespect** [RocktimSaikia/anime-chan#28]
and if there's any way to access all the characters names :+1:

**Comment by RocktimSaikia** [comment]
Yes. We can do that and I even thought about the execution before but that would require a lot of time to execute plus it would need a dedicated database that supports media files like [`Cloudinary`](https://cloudinary.com/) which has different pricing plans and since this is just an open-source project I don't want to spend a few extra bucks on adding that feature. Unless someone is willing to back it financially.

**Comment by RocktimSaikia** [comment]
@AndresMorelos  That would still require a lot of time. To get the images first then linking to their corresponding characters.  So I don't plan on investing time in adding this feature for now. Although I never tried it with MongoDB before I am sure it would not optimize the images. I only prefer MongoDB for text-based data storage, not for media storage.  Maybe in the future, If I ever get time I will think about it.

**Comment by quiktea** [comment]
@RocktimSaikia why don't you add the images as some sort of link. Like imgur links? If users every needed to extract image data, they could do so within their own code.

**Comment by boneskewer69** [comment]
For a personal project I am using the AniList API. I'm taking the character name I get of a quote by animechan and form a simple GraphQL query.

This is what my query looks like:

```
{
  Character(search: "Levi") {
    name {
      first
      last
      full
      native
    }
    image {
      large
    }
    #description
}
```

Inside the response I get a link to an image of the character. In this case: `https://s4.anilist.co/file/anilistcdn/character/large/b45627-CR68RyZmddGG.png`

Maybe this can help you @ThanksILoveYouWithAllTheRespect.

**Comment by MokiyCodes** [comment]
just thought copyright law might be worth mentioning, as you'd need permission from each studio/publisher to use the image(s) in question

**Comment by Nekidev** [comment]
> just thought copyright law might be worth mentioning, as you'd need permission from each studio/publisher to use the image(s) in question, to be legally allowed to serve it to the end user(s)

I don't think studios would bother about having an API that shows their info. Do you really think that MAL/AL/Kitsu/etc ask every studio if they can use the cover?

**Comment by galpt** [comment]
> Yes. We can do that and I even thought about the execution before but that would require a lot of time to execute plus it would need a dedicated database that supports media files like [`Cloudinary`](https://cloudinary.com/) which has different pricing plans and since this is just an open-source project I don't want to spend a few extra bucks on adding that feature. Unless someone is willing to back it financially.

@rocktimsaikia  If people are still hoping for this, I guess our group could back this up without burning so much cash.
Like I said, we're willing to back it up and see how it goes.
If the images don't have copyright issues or just the cover images, looking at MAL, I don't think it's impossible and studios wouldn't mind people seeing cover images, especially if that will give them more exposure to the shows they made.

We even run some other services (e.g. mirrors) & succeeded in handling at least 2m+ requests in the last 30 days with almost no problem.
Supporting this project shouldn't be much of a problem. :)
In case anyone's interested, I'm available on Discord `!Castellalala~#0218`.
